### DISCUSSION TITLE
How to get nuget not try to access my `~/.nuget`?

### DISCUSSION THREAD

**Discussion by WhiteBlackGoose** [NuGet/Home#12587] [Q&A] [answered]
Hello!

I'm trying to restore packages in my project. However, only CWD is accessible, not `~/.nuget`. So restoration fails with 
```
error NU1301: Unable to load the service index for source https://api.nuget.org/v3/index.json
```

I run `dotnet restore` and I have `nuget.config` in CWD. Any idea how I can disable caching in `~/.nuget`?

**Comment by baronfel** [comment, marked as answer]
@WhiteBlackGoose a few ways: https://learn.microsoft.com/en-us/nuget/consume-packages/managing-the-global-packages-and-cache-folders

>* Override using the NUGET_PACKAGES environment variable, 
>* the `globalPackagesFolder` or `repositoryPath` [configuration settings](https://learn.microsoft.com/en-us/nuget/reference/nuget-config-file#config-section) (when using PackageReference and packages.config, respectively), or the `RestorePackagesPath` MSBuild property (MSBuild only). The environment variable takes precedence over the configuration setting.

**Reply by WhiteBlackGoose** [reply]
Thanks. Didn't help my problem sadly, but answers the question
